### DISCUSSION TITLE
Deployment aka Production environment

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#286] [General]
We can add anything that might be useful for the production environment setup and deployment.

**Comment by almirsarajcic** [comment]
Usually, when running `mix phx.gen.release` (with or without the `--docker` flag), file `release.ex` gets created with code for running migrations in Elixir releases, but it seems Ash requires a bit different file. So, here it is:
https://hexdocs.pm/ash_postgres/migrations-and-tasks.html#running-migrations-in-production

**Comment by fradarr** [comment]
Looking forward having a full blown CD/CI workflow
